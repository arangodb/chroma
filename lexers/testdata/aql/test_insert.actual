INSERT {
    "name": "Ned",
    "surname": "Stark",
    "alive": true,
    "age": 41,
    "traits": ["A","H","C","N","P"]
} INTO Characters