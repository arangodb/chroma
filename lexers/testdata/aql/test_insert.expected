[
  {"type":"KeywordReserved","value":"INSERT"},
  {"type":"Text","value":" "},
  {"type":"Punctuation","value":"{"},
  {"type":"Text","value":"\n    "},
  {"type":"LiteralStringDouble","value":"\"name\""},
  {"type":"Operator","value":":"},
  {"type":"Text","value":" "},
  {"type":"LiteralStringDouble","value":"\"Ned\""},
  {"type":"Punctuation","value":","},
  {"type":"Text","value":"\n    "},
  {"type":"LiteralStringDouble","value":"\"surname\""},
  {"type":"Operator","value":":"},
  {"type":"Text","value":" "},
  {"type":"LiteralStringDouble","value":"\"Stark\""},
  {"type":"Punctuation","value":","},
  {"type":"Text","value":"\n    "},
  {"type":"LiteralStringDouble","value":"\"alive\""},
  {"type":"Operator","value":":"},
  {"type":"Text","value":" "},
  {"type":"KeywordConstant","value":"true"},
  {"type":"Punctuation","value":","},
  {"type":"Text","value":"\n    "},
  {"type":"LiteralStringDouble","value":"\"age\""},
  {"type":"Operator","value":":"},
  {"type":"Text","value":" "},
  {"type":"LiteralNumberFloat","value":"41"},
  {"type":"Punctuation","value":","},
  {"type":"Text","value":"\n    "},
  {"type":"LiteralStringDouble","value":"\"traits\""},
  {"type":"Operator","value":":"},
  {"type":"Text","value":" "},
  {"type":"Punctuation","value":"["},
  {"type":"LiteralStringDouble","value":"\"A\""},
  {"type":"Punctuation","value":","},
  {"type":"LiteralStringDouble","value":"\"H\""},
  {"type":"Punctuation","value":","},
  {"type":"LiteralStringDouble","value":"\"C\""},
  {"type":"Punctuation","value":","},
  {"type":"LiteralStringDouble","value":"\"N\""},
  {"type":"Punctuation","value":","},
  {"type":"LiteralStringDouble","value":"\"P\""},
  {"type":"Punctuation","value":"]"},
  {"type":"Text","value":"\n"},
  {"type":"Punctuation","value":"}"},
  {"type":"Text","value":" "},
  {"type":"KeywordReserved","value":"INTO"},
  {"type":"Text","value":" "},
  {"type":"Name","value":"Characters"}
]
